### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by clappedev** [3kh0/ext-remover#211] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm on Chrome OS version 102. The ext remover has been working perfectly fine for me but after I did chrome://restart I can go to the menu to disable extensions but they don't actually disable. Any help her?

**Comment by bartholemuew** [comment, marked as answer]
Your chromebook might've been updated and the exploit got patched.

**Reply by clappedev** [reply]
It hasn't been updated.

**Reply by Ekanath-K** [reply]
Yep its just patched on like 102 and 106

**Reply by clappedev** [reply]
it makes no sense that it would ONLY be patched on 102 not 103 or 104

**Reply by Ekanath-K** [reply]
I am not sure, my friend's version is 105 and it works.
Mine is 106 so it doesn't work. 
I haven't seen any below 104 before sorry.

**Reply by Fallenmoon8080** [reply]
Mine is on 102 and it works just fine.

**Reply by clappedev** [reply]
Are you guys on 102 LTS? Because its long term support so it still gets updates

**Reply by epicgame12414** [reply]
i believe the reason it doesn't work on 102 on some computers is that there are different Chromebook models i have a new model and a few friends have old models we have SAME update/OS version but doesn't work on mine
